package.json

```diff
@@ -12,7 +12,9 @@
     "bot:run": "node dist/start.js",
     "test:all": "mocha",
     "test:coverage": "c8 --reporter html --reporter text --all --src ./src mocha",
-    "test": "yarn test:coverage"
+    "test": "yarn test:coverage",
+    "docker:up:staging": "docker compose --env-file .env.staging up --build bot-dev",
```

Comment:
I would prefer not to implement this pattern. In the application, the runner is docker and npm has environment-agnostic tasks. With the change, we have npm running docker to run npm again, making not so evident the execution flow. Also the `.env.staging` file does not exist in other environments.

Reply:
Ok, moved to shell scripts